src/utils/automation-helper.js

```diff
@@ -32,6 +32,9 @@ export const getAutomation = async (automationId) => {
 
 export const getAutomationRun = async (automationId, runId) => {
   const response = await fetch(`../api/v1/automations/${automationId}/runs/${runId}`);
+  if (response.status !== 200) {
+    throw Error;
```

Comment:
You are throwing the `Error` function here. Sort of like `throw () => {}`.

Consider something like `throw new Error()` instead.

Reply:
Fixed this in the latest commit